caffe2/operators/hip/conv_op_miopen.hip

```diff
@@ -49,17 +49,15 @@ class MIOPENConvOpBase : public ConvPoolOpBase<HIPContext> {
     MIOPEN_ENFORCE(miopenCreateTensorDescriptor(&top_desc_for_bias_));
     MIOPEN_ENFORCE(miopenCreateConvolutionDescriptor(&conv_desc_));
 
-    if (group_ > 1) {
+    if ((operator_def.type().substr(0, 15) == "DepthWiseConv") ||
```

Comment:
We are moving away from using OperatorDef. You can instead add a template parameter 
```
typename <bool DepthWise>
class MIOPENConvOpBase : ... 
```
and then set `mode_` based on `DepthWise`